api/proto/teleport/okta/v1/okta_service.proto

```diff
@@ -51,8 +51,166 @@ service OktaService {
   rpc DeleteOktaAssignment(DeleteOktaAssignmentRequest) returns (google.protobuf.Empty);
   // DeleteAllOktaAssignments removes all Okta assignments.
   rpc DeleteAllOktaAssignments(DeleteAllOktaAssignmentsRequest) returns (google.protobuf.Empty);
+
+  // ValidateClientCredentials checks if the provided client credentials are valid.
+  rpc ValidateClientCredentials(ValidateClientCredentialsRequest) returns (ValidateClientCredentialsResponse);
+  // EnrollIntegration enrolls a new integration with the specified parameters.
+  rpc EnrollIntegration(EnrollIntegrationRequest) returns (EnrollIntegrationResponse);
```

Comment:
We don't normally use "enroll" terminology in our API. Let's just stick to standard "create"?

Reply:
We don’t use it in the code, but in our conversations, we refer to it as the integration/plugin enrollment flow. My reasoning was that the API should reflect the business domain, but from a developer’s perspective, CreateIntegration is more meaningful.

I don't have strong feeling on that so I have  Changed it to `CreateIntegration`